src/parser/jobs/sch/Weaving.js

```diff
@@ -0,0 +1,84 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS, {getAction} from 'data/ACTIONS'
+import CoreWeaving from 'parser/core/modules/Weaving'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const WEAVING_SEVERITY = {
+	1: SEVERITY.MINOR,
+	5: SEVERITY.MEDIUM,
+	10: SEVERITY.MAJOR,
+}
+
+export default class Weaving extends CoreWeaving {
+	static handle = 'weaving'
```

Comment:
`handle` and `title` are defined in the parent, no need to define here.

Do need to define the entire dependencies array though, which is a pain. I'll have a look into allowing subclasses to add to dependencies w/o copying the entire thing.